MVVM Module/Tests/Feed UI/Helpers/FeedViewController+TestHelpers.swift

```diff
@@ -57,4 +61,12 @@ extension FeedViewController {
 	private var feedImagesSection: Int {
 		return 0
 	}
+
+	var feedErrorView: ErrorView? {
+		return tableView.tableHeaderView as? ErrorView
+	}
+
+	var errorMessage: String? {
+		return feedErrorView?.message
+	}
```

Comment:
Following my previous comment `feedErrorView` can be removed

```suggestion
	var errorMessage: String? {
		return errorView?.message
	}
```